config/filterMap.py

```diff
@@ -0,0 +1,33 @@
+# This file is part of faro.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+
+from lsst.faro.utils.filter_map import FilterMapDict
+
+config.data = {
+    "hsc*": FilterMapDict(data = {
+        "g": "HSC-G",
+        "r": "HSC-R",
+        "i": "HSC-I",
```

Comment:
Need to account for the fact that HSC r- and i-band filters can be _either_ "HSC-R" or "HSC-R2" and "HSC-I" or "HSC-I2"?

Reply:
Hmm, I think we would need to know which filter was used as the reference for calibration, and assume that appropriate chromatic corrections have been applied to the stars. That sounds like a bigger issue that we will need to resolve.